AIMSICD/src/main/AndroidManifest.xml

```diff
@@ -65,6 +70,11 @@
                     android:name="android.support.PARENT_ACTIVITY"
                     android:value=".AIMSICD"/>
         </activity>
+        
+        <activity
+                    android:name="com.bugsnag.android.API_KEY"
+                    android:value="9653f64c8477152cc19eb9efb011397d"/>
```

Comment:
Isn't that what is required to be done [here](https://github.com/freefair/advanced-bugsnag-android#configuring-your-androidmanifest)?